bitnami/wordpress-nginx/6/debian-11/rootfs/opt/bitnami/scripts/libwordpress.sh

```diff
@@ -228,7 +228,7 @@ wordpress_initialize() {
 
     # Check if WordPress has already been initialized and persisted in a previous run
     local -r app_name="wordpress"
-    if ! is_app_initialized "$app_name" || [[ ! -f "$WORDPRESS_CONF_FILE" ]]; then
+    if ! is_app_initialized "$app_name" || [[ ! -f "${BITNAMI_VOLUME_DIR}/${app_name}/wp-config.php" ]]; then
```

Comment:
Hi, sorry about the delay! 

I agree that checking `$WORDPRESS_CONF_FILE` is not helpful as we are generating it in the postunpack stage and the correct check is the one you added. That said, I think we could remove the first condition of this `if` as it is only checking if `${BITNAMI_VOLUME_DIR}/${app_name}` is empty (which is implicitly done by the second condition):

https://github.com/bitnami/containers/blob/b3ea0b2b21484e5184c0acee0fa61228d591367a/bitnami/wordpress-intel/6/debian-11/prebuildfs/opt/bitnami/scripts/libpersistence.sh#L114